packages/cli/core/src/commands/org/resources/pull.ts

```diff
@@ -75,17 +75,15 @@ export default class Pull extends CLICommand {
       description: 'Overwrite resources directory if it exists.',
       default: false,
     }),
-    resourceTypes: Flags.build<ResourceSnapshotType>({
-      parse: async (resourceType: ResourceSnapshotType) => resourceType,
-    })({
+    resourceTypes: Flags.enum<ResourceSnapshotType>({
```

Comment:
`Flags.build` has been deprecated, here `enum` seems the most adapted.

Reply:
not sure why this is past of this PR

Reply:
My own comments on the flags are outdated, I did fix that in another PR in the end. (that's why it's marked as outdated)